.github/dependabot.yml

```diff
@@ -9,6 +9,8 @@ updates:
       - "Source: Internal"
       - "dependencies"
     open-pull-requests-limit: 100
+    commit-message:
+      prefix: "[dependency]"
```

Comment:
What would the PR title look like?

I am not in favour of this, IMO `Bump abc ... from X to Y` is clear enough.

Reply:
I can try just blank prefix, so it will stay "bump ..." instead of "[dependency] Bump...". idk if that will work (docs are not very specific of what values are allowed)